arm/Microsoft.RecoveryServices/vaults/protectionContainers/protectedItems/deploy.bicep

```diff
@@ -1,66 +0,0 @@
-@description('Required. Name of the resource.')
```

Comment:
Hey @prasanjeets , why did you remove so many files? Especially regarding protected items?

Reply:
Corrected